examples/src/main/java/com/mapbox/navigation/examples/core/BasicNavigationActivity.kt

```diff
@@ -134,6 +134,7 @@ open class BasicNavigationActivity : AppCompatActivity(), OnMapReadyCallback {
                         .accessToken(Utils.getMapboxAccessToken(applicationContext))
                         .coordinates(originLocation.toPoint(), null, latLng.toPoint())
                         .alternatives(true)
+                        .annotationsList(mutableListOf("maxspeed", "duration", "congestion"))
```

Comment:
Are these 3 needed for getting speed limit data (it seems that at least `maxspeed`)? If so, are we documenting that? If not, it'd be great to add a comment in the `MapMatcherResult` `speedLimit` docs.

Reply:
We only need `maxspeed`. Good catch. I will remove the other two annotations.